README.md

```diff
@@ -840,17 +840,15 @@ string, please try the [paper_trail-globalid][37] gem.
 
 ### 4.b. Associations
 
-Discussed as early as 2009, and first implemented in late 2014, association
-tracking was maintained until 2018 as an experimental feature, not recommended
-for production. During that time a steady stream of issues were reported, there
-were not enough volunteers fixing them, and the list of known issues grew. In
-2018, the feature was moved to a separate gem,
-[paper_trail-association_tracking][6].
-
-To avoid breaking changes, `paper_trail` will have a runtime dependency on
-`paper_trail-association_tracking` and keep running the existing tests related
-to association tracking. This arrangement will be maintained for a few years, if
-practical.
+If you would like to track and reify associations you can utilize the following PT plugin: [paper_trail-association_tracking][6]
+
+A little history lesson, discussed as early as 2009, and first implemented in late 2014, association
+tracking was part of PT core until 2018 as an experimental feature and was use at your own risk. The gem now
+maintains a list of known issues and we hope the community can help remove some of them via PR's.
```

Comment:
What was your goal with this paragraph? I think I prefer the existing version.

Reply:
I think the tone has improved

Reply:
I agree the existing version sounds a bit negative. I was trying to explain why the gem was extracted, that's why I had that negative-sounding focus on the list of known issues. Maybe we could go with what you have but also direct people to https://github.com/paper-trail-gem/paper_trail/issues/1070 if they want to know why it happened?

Reply:
Sure just write something up that you like. I mainly was concerned about the tone thats all.